test/test_recurrent_layer.h

```diff
@@ -95,11 +95,11 @@ TEST(recurrent_cell, train_different_batches) {
   }
 }
 
-TEST(recurrent_cell, train2) {
+TEST(recurrent, train2) {
   network<sequential> nn;
   gradient_descent optimizer;
 
-  nn << recurrent_cell_layer(4, 6) << tanh() << recurrent_cell_layer(6, 3)
+  nn << recurrent_layer(rnn(4, 6),1) << tanh() << recurrent_layer(rnn(6, 3),1)
```

Comment:
Exact, now the cell must be wrapped since having a raw cell (single time step) makes no sense.